Commands/ImportGA4Reports.php

```diff
@@ -72,6 +73,8 @@ protected function executeImpl(): int
         $timezone = $input->getOption('timezone');
         $extraCustomDimensions = $this->getExtraCustomDimensions();
         $property = $input->getOption('property');
+        $streamId = $input->getOption('streamId');
+        $streamIDs = $streamId ? explode(',', $streamId) : [];
```

Comment:
No need to change it because it's not a big deal, but the case is a bit inconsistent. Most places use `streamIds` instead of `streamIDs`

Reply:
made it consistent